.pipelines/cni/lsg/kernel-upgrade-template.yaml

```diff
@@ -0,0 +1,307 @@
+              inlineScript: |
+                make -C ./hack/aks ${{ parameters.clusterType }} \
+                AZCLI=az REGION=$(LOCATION) SUB=$(SUB_AZURE_NETWORK_AGENT_BUILD_VALIDATIONS) \
+                CLUSTER=${{ parameters.clusterName }}-$(commitID) \
+                VM_SIZE=${{ parameters.vmSize }} \
+                AUTOUPGRADE=none
+            name: "CreateAksCluster"
+            displayName: "Create AKS Cluster"
+
+  - stage: ${{ parameters.name }}
+    variables:
+      commitID: $[ stagedependencies.setup.env.outputs['SetEnvVars.commitID'] ]
+      cnsVersion: $[ stagedependencies.setup.env.outputs['SetEnvVars.cnsVersion'] ]
+      cniVersion: $[ stagedependencies.setup.env.outputs['SetEnvVars.cniVersion'] ]
+    pool:
+      name: $(BUILD_POOL_NAME_DEFAULT)
+    dependsOn:
+      - create_${{ parameters.name }}
+      - setup
+    displayName: "CNIv2 Test - ${{ parameters.name }}"
+    jobs:
+      - job: integration
+        displayName: "Integration Test - ${{ parameters.name }}"
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                echo cns version - $(cnsVersion)
+                echo cni version - $(cniVersion)
+
+                echo "Start Integration Tests on Overlay Cluster"
+                make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(commitID)
+                kubectl cluster-info
+                kubectl get po -owide -A
+                sudo -E env "PATH=$PATH" make test-integration CNS_VERSION=$(cnsVersion) CNI_VERSION=$(cniVersion) INSTALL_CNS=true INSTALL_AZURE_CNI_OVERLAY=true CNS_IMAGE_REPO=$(CNS_IMAGE_REPO) CNI_IMAGE_REPO=$(CNI_IMAGE_REPO)
+            name: "overlaye2e"
+            displayName: "Overlay Integration"
+            retryCountOnTaskFailure: 2
+      - job: kernel_upgrade
+        displayName: "Kernel Node Upgrade"
+        dependsOn: integration
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                set -ex
+                az extension add --name aks-preview
+                make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(commitID)
+            retryCountOnTaskFailure: 3
+            name: "config"
+            displayName: "Set Kubeconfig"
+          - script: |
+              set -e
+
+              echo "-- Start privileged daemonset --"
+              kubectl get pods -Aowide
+              kubectl apply -f test/integration/manifests/load/privileged-daemonset.yaml
+              sleep 10s
```

Comment:
Why sleep ?

Reply:
Holdover from the bash script. Will remove.